app/controllers/organizations/adoption_application_reviews_controller.rb

```diff
@@ -1,9 +1,15 @@
 class Organizations::AdoptionApplicationReviewsController < Organizations::BaseController
-  before_action :active_staff
+  before_action :set_adopter_application, only: %i[edit update]
+
   layout "dashboard"
 
   def index
-    @q = Pet.org_pets_with_apps(current_user.staff_account.organization_id).ransack(params[:q])
+    authorize! AdopterApplication,
+      context: {organization: current_user.organization}
```

Comment:
Just noticed this - should we have a `before_action authenticate_user` or do this in case someone tries to access this page without logging in?
`context: {organization: Current.organization}`

Reply:
You are right! That should have been Current.organization and was def a mistake.